modules/gdnative/gdnative_api.json

```diff
@@ -257,6 +257,13 @@
             ["const godot_object *", "p_object"]
           ]
         },
+        {
```

Comment:
I think 3.1 comes with the core 1.1 extension, so a new extension would have to be made for this.

Reply:
The 1.2 extension has been created, so it can be used for this PR.